hadoop-hdds/framework/src/main/java/org/apache/hadoop/hdds/utils/db/RocksDBConfiguration.java

```diff
@@ -46,6 +46,20 @@ public class RocksDBConfiguration {
       description = "OM RocksDB logging level (INFO/DEBUG/WARN/ERROR/FATAL)")
   private String rocksdbLogLevel;
 
+  @Config(key = "rocksdb.max.log.file.size",
+      type = ConfigType.SIZE,
+      defaultValue = "0MB",
+      tags = {OM, SCM, DATANODE},
+      description = "Maximum size of RocksDB application log file.")
+  private long rocksdbMaxLogFileSize = 0;
+
+  @Config(key = "rocksdb.keep.log.file.num",
+      type = ConfigType.INT,
+      defaultValue = "1000",
+      tags = {OM, SCM, DATANODE},
+      description = "Maximum number of RocksDB application log files.")
+  private int rocksdbKeepLogFileNum = 1000;
```

Comment:
Is 1000 the current default?

Reply:
Yes, [keep_log_file_num](https://github.com/facebook/rocksdb/blob/main/include/rocksdb/options.h#L874) as mentioned in the description.